docs/reference/query-dsl/prefix-query.asciidoc

```diff
@@ -1,33 +1,40 @@
 [[query-dsl-prefix-query]]
 === Prefix Query
+Returns documents that contain a specific prefix in a provided field.
 
-Matches documents that have fields containing terms with a specified
-prefix (*not analyzed*). The prefix query maps to Lucene `PrefixQuery`.
-The following matches documents where the user field contains a term
-that starts with `ki`:
+[[prefix-query-ex-request]]
+==== Example request
+
+The following search returns documents where the `user` field contains a term
+that begins with `ki`.
 
 [source,js]
---------------------------------------------------
+----
 GET /_search
-{ "query": {
-    "prefix" : { "user" : "ki" }
-  }
+{
+    "query": {
+        "prefix": {
+            "user": {
+                "value": "ki",
+                "rewrite": "constant_score"
```

Comment:
I'm wondering if this parameter should be included in the "default" example since IMHO its pretty much an advanced parameter and mentioned later (also, `constant_score` should be the default, so not necessary here).
Did you consider also documenting the previous "short" form or do we want to discourage ppl. from using it?

Reply:
Great question!

I've been using the longer form requests because it's easier to delineate additional parameters. This is less of an issue with the `prefix` query, but I used the longer form so it's consistent with other queries. However, I'm open to changing this if wanted.

I've removed the  `rewrite ` parameter and added a "short request example" with 1e6ca49 and d58de59.